### PR TITLE
Star the message and display

### PR DESCRIPTION
**Pull request by SumanthVenkata**

### CODE REVIEW

File: src/app/Component/Common/Header.css [hunk 1]
```diff
@@ -105,3 +105,67 @@
...
+    right: 10%;
+    background: #50535a;
+    box-shadow: 0px 0px 30px rgb(14 10 16 / 84%);
+    border-radius: 10px;
+}
+.header-star-img{
+    display: flex;
+}
+.header-star-user{
+    margin: 5px;
```
Comment (harish-balagoni) [inline src/app/Component/Common/Header.css:123]: need explanation for this change

Comment (SumanthVenkata) [reply src/app/Component/Common/Header.css:123]: I added a another option called star messages in options.
To satisfy the four options height and width I made changes

Comment (harish-balagoni) [reply src/app/Component/Common/Header.css:123]: Why do we need margin: 5px here?

Comment (SumanthVenkata) [reply src/app/Component/Common/Header.css:123]: i used to display user profile pic and user name in same line. username is not aligned properly thats why i used margin 5 px to align the username and user pic align in same line propely

**Pull request closed without merging**